### ISSUE TITLE
Missing Microsoft.VisualStudio.Web.CodeGeneration.Design::8.0.0-preview.7 on NuGet

### ISSUE THREAD

**Issue by jasontaylordev** [dotnet/Scaffolding#2493]
The Microsoft.VisualStudio.Web.CodeGeneration.Design::8.0.0-preview.7 version is not available on NuGet.

https://www.nuget.org/packages/Microsoft.VisualStudio.Web.CodeGeneration.Design

The latest version is preview6, which is incompatible with preview7.

**Comment by jasontaylordev** [comment]
@nguyentuananh921 it’s the same for me. It’s because they didn’t publish microsoft.visualstudio.web.codegeneration.design::8.0.0-preview.7. So instead preview.6 gets installed which has version conflicts with other preview.7 packages such as EF Core.

**Comment by nguyentuananh921** [comment]
> @nguyentuananh921 it’s the same for me. It’s because they didn’t publish microsoft.visualstudio.web.codegeneration.design::8.0.0-preview.7. So instead preview.6 gets installed which has version conflicts with other preview.7 packages such as EF Core.

So What should I do now? I am stucking with creating Identity. Should I downgrade all package to learn more instead of waiting sir?

**Comment by jasontaylordev** [comment]
> So What should I do now? I am stucking with creating Identity. Should I downgrade all package to learn more instead of waiting sir?

There are a few options. 

1. Uninstall .NET SDK 8 preview 7 and reinstall preview 6. 
2. Keep .NET 8 preview 7, but rollback all NuGet packages to preview 6. 
3. Create a new project that targets preview 6, scaffold, and then move the scaffolded code to your preview 7 project.  
4. Wait for them to properly release the new package. 

I went with option 2. Once I had finished scaffolding, I removed the scaffolding packages and upgraded the remaining packages to preview 7.
